notion/assets/dashboards/NotionDashboard.json

```diff
@@ -0,0 +1 @@
_span":"1w"},"type":"query_value","requests":[{"response_format":"scalar","queries":[{"data_source":"logs","name":"query1","indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@event.timestamp"},"group_by":[],"search":{"query":"@event.type:page.permissions.shared_to_public_role_added"},"storage":"hot"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":0},"layout":{"x":6,"y":0,"width":3,"height":2}},{"id":1811418849110210,"definition":{"title":"Pages Shared to Web - Past 24 Hours","title_size":"16","title_align":"left","time":{"live_span":"1d"},"type":"query_value","requests":[{"response_format":"scalar","queries":[{"data_source":"logs","name":"query1","indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@event.timestamp"},"group_by":[],"search":{"query":"@event.type:page.permissions.shared_to_public_role_added"},"storage":"hot"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":0},"layout":{"x":9,"y":0,"width":3,"height":2}},{"id":6351406053635982,"definition":{"title":"Pages shared with guests - Past Month","title_size":"16","title_align":"left","show_legend":false,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1mo"},"type":"timeseries","requests":[{"formulas":[{"alias":"Pages Shared with guests","formula":"query1"}],"queries":[{"data_source":"logs","name":"query1","indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@event.timestamp"},"group_by":[],"search":{"query":"@event.type:page.permissions.guest_role_added"},"storage":"hot"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":0,"y":2,"width":6,"height":2}},{"id":658320621850924,"definition":{"title":"Pages shared with guests - Past Week","title_size":"16","title_align":"left","time":{"live_span":"1w"},"type":"query_value","requests":[{"response_format":"scalar","queries":[{"data_source":"logs","name":"query1","indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@event.timestamp"},"group_by":[],"search":{"query":"@event.type:page.permissions.guest_role_added"},"storage":"hot"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":0},"layout":{"x":6,"y":2,"width":3,"height":2}},{"id":3232763984257404,"definition":{"title":"Pages shared with guests - Past 24 Hours","title_size":"16","title_align":"left","time":{"live_span":"1d"},"type":"query_value","requests":[{"response_format":"scalar","queries":[{"data_source":"logs","name":"query1","indexes":["*"],"compute":{"aggregation":"cardinality","metric":"@event.timestamp"},"group_by":[],"search":{"query":"@event.type:page.permissions.guest_role_added"},"storage":"hot"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":0},"layout":{"x":9,"y":2,"width":3,"height":2}}]},"layout":{"x":0,"y":5,"width":12,"height":1}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
Could you format this JSON using an online tool or a CLI tool like `jq`? It will make both the review of this PR easier, and it will make future PR "diffs" smaller and easier to review. Thanks!

Reply:
done ✅

Reply:
Thanks, looks good!